Mathlib/GroupTheory/GroupAction/Opposite.lean

```diff
@@ -92,6 +101,53 @@ instance Mul.toHasOppositeSMul [Mul α] : SMul αᵐᵒᵖ α :=
 #align has_mul.to_has_opposite_smul Mul.toHasOppositeSMul
 #align has_add.to_has_opposite_vadd Add.toHasOppositeVAdd
 
+namespace RightActions
+
+/-- With `open scoped RightActions`, an alternative symbol for left actions, `r • m`.
+
+In lemma names this is still called `smul`. -/
+scoped notation3:74 r:75 " •> " m:74 => r • m
+
+/-- With `open scoped RightActions`, a shorthand for right actions, `op r • m`.
+
+In lemma names this is still called `op_smul`. -/
+scoped notation3:73 m:73 " <• " r:74 => MulOpposite.op r • m
+
+/-- With `open scoped RightActions`, an alternative symbol for left actions, `r • m`.
+
+In lemma names this is still called `vadd`. -/
+scoped notation3:74 r:75 " +ᵥ>  " m:74 => r +ᵥ m
```

Comment:
```suggestion
scoped notation3:74 r:75 " +ᵥ> " m:74 => r +ᵥ m
```
(just style, not a functional change)

Reply:
This one _is_ functional, isn't it? It removes an unwanted space from the pretty-printed output?

Reply:
I'm pretty sure that the pretty printer only uses that there is a space as a hint to insert a single space.